rewriter/src/test/java/com/facebook/coresql/rewriter/TestOrderByRewriter.java

```diff
@@ -70,29 +75,34 @@
 
     private void assertOrderByPatternIsMatchedOrUnmatched(String sql, boolean isMatched)
     {
+        AstNode ast = parseStatement(sql);
```

Comment:
In general a function taking a boolean parameter and using that as a flag is a good sign that you should have 2 functions instead. It probably makes more sense to have `assertQueryRewritten(string original, string expected)` and `assertQueryNotChanged(string original)`.

Reply:
Refactored the test to be in the form you suggested.

Done